site/public/css/simple-grading.css

```diff
@@ -29,6 +29,49 @@ input[type="number"]::-webkit-outer-spin-button {
     display: none !important;
 }
 
+.scrollable-table{
+    overflow-x: auto;
+}
+th{
+    background: var(--standard-light-gray);
+}
+
+tr>th:first-child,tr>td:first-child
+{
+    position: sticky;
+    left: 0rem;
+    border-left: 1px solid var(--standard-medium-gray);
+  }
+tr>th:nth-child(2),tr>td:nth-child(2)
+{
+    position: sticky;
+    left: 1.25rem;
+  }
+tr>th:nth-child(3),tr>td:nth-child(3)
+{
+    position: sticky;
+    left: 5.25rem;
+  }
+tr>th:nth-child(4),tr>td:nth-child(4)
+{
+    position: sticky;
+    left: 10.75rem;
+  }
+tr>th:nth-child(5),tr>td:nth-child(5)
+ {
+    position: sticky;
+    left: 16.75rem;
+    border-right: 1px solid var(--standard-medium-gray);
+  }
+  
+  tr:nth-child(odd) td {
+    background: white;
```

Comment:
Should this be fixed to white, or should it be changed to a variable listed in `colors.css` like the ones used elsewhere in these changes?